content/applications/hr/payroll/contracts.rst

```diff
@@ -65,41 +62,52 @@ Required fields
 Optional fields
 ---------------
 
-- :guilabel:`Employee`: Name of the employee that the contract applies to.
-- :guilabel:`Department`: The department the contract applies to.
-- :guilabel:`Job Position`: The specific job position the contract applies to.
-- :guilabel:`Contract Type`: Choose from :guilabel:`CDI`, :guilabel:`CDD`, or :guilabel:`PFI` from
+- :guilabel:`Employee`: name of the employee that the contract applies to.
+- :guilabel:`Department`: the department the contract applies to.
+- :guilabel:`Job Position`: the specific job position the contract applies to.
+- :guilabel:`Contract Type`: choose from :guilabel:`CDI`, :guilabel:`CDD`, or :guilabel:`PFI` from
   the drop-down menu.
 
   - :guilabel:`CDI` is an open-ended contract with only a start date but no end date.
   - :guilabel:`CDD` is a contract with both a start date and an end date.
   - :guilabel:`PFI` is a Belgian-specific contract used when hiring employees that need training,
     and covers the training period specifically.
 
-- :guilabel:`End Date`: If the contract has a specific end date, click the drop-down menu, navigate
+- :guilabel:`End Date`: if the contract has a specific end date, click the drop-down menu, navigate
   to the correct month and year using the arrow icons, then click on the date.
```

Comment:
do you want to implement those same arrow icons that you used previously? that might be a nice touch here.